### ISSUE TITLE
[editor] additional weather option

### ISSUE THREAD

**Issue by SputNikPlop** [muxable/rtirl-obs#270]
Overlay for both Celsius and Fahrenheit separately by a / or something. It’s easy enough for us to display two separate sources but I really like to make everything aligned nicely and it’s hard to do when the temperatures take up different space depending on the values and also font. So having them as a combined source would be really useful.

**Comment by kevmo314** [comment]
I don't think this would work like streamers would expect. the `/` would move around a lot based on the width of the first temperature. Honestly I think two separate sources is a better solution. If we provide an overlay that merges the two I think most people would realize they prefer it separated so they can control the alignment.

It might be worth providing an example or something though, I suspect the idea of like a news ticker type feed of overlays would be interesting to people but it would require custom code. Maybe we can make an examples directory and put some more complex overlays there that aren't surfaced in the editor.

**Comment by SputNikPlop** [comment]
> Maybe we can make an examples directory and put some more complex overlays there that aren't surfaced in the editor.

This would work, this is for Tazo that requested this
